packages/libs/wdk-client/src/Components/Display/CollapsibleDetailsSection.tsx

```diff
@@ -1,17 +1,18 @@
-import React, { ReactNode, useState } from 'react';
+import React, { ReactNode } from 'react';
 
 interface Props {
   summary: ReactNode;
   collapsibleDetails: ReactNode;
-  initialShowDetailsState?: boolean;
+  showDetails: boolean;
+  setShowDetails: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
Can we change this to the following? I feel like the current type is an implementation detail.

```suggestion
  setShowDetails: (showDetails: boolean) => void;
```